docs/introduction/roadmap.md

```diff
@@ -1 +1,6 @@
+---
+hide:
+- toc
+---
+
 ![Roadmap](../img/roadmap/miden-roadmap-dec-23.jpg)
```

Comment:
Can you maybe use the new Roadmap? 

![Miden Roadmap](https://github.com/0xPolygonMiden/miden-base/assets/35031754/ff75b697-7831-4669-bdf4-b25840f8455e)

Reply:
just need to get hold of this file